src/encodings/decoding.rs

```diff
@@ -328,7 +328,7 @@ pub struct DeltaBitPackDecoder<T: DataType> {
 
 impl<T: DataType> DeltaBitPackDecoder<T> {
   pub fn new() -> Self {
-    Self { bit_reader: None, num_values: 0, num_mini_blocks: 0,
+    Self { bit_reader: BitReader::from(vec![]), num_values: 0, num_mini_blocks: 0,
```

Comment:
I replaced Option<BitReader> with just BitReader, because it fixes TODO item and removes assertions on whether or not bit reader is already set. I think that this is equivalent of bit reader being empty (if `set_data` is not called).

Reply:
My understanding was that the code will fail anyway, when `set_data()` is not called, because of the reading from empty bit reader. I can add boolean flag to check if `set_data()` was called and assert it in `get()`.